manifests/plugin/rh_cloud.pp

```diff
@@ -1,5 +1,15 @@
-# Installs rh_cloud plugin
-class foreman::plugin::rh_cloud {
+# @summary Installs rh_cloud plugin
+#
+# @param enable_iop_advisor_engine
+#   Enable iop-advisor-engine integration
+#
+class foreman::plugin::rh_cloud (
+  Boolean $enable_iop_advisor_engine = false,
+) {
   foreman::plugin { 'rh_cloud':
   }
+
+  if $enable_iop_advisor_engine {
+    include iop_advisor_engine
+  }
```

Comment:
```suggestion
  class { 'iop_advisor_engine':
    ensure => bool2str($enable_iop_advisor_engine, 'present', 'absent')
  }
```
like that?

Reply:
If we can implement that then I do think it is nice for users.

Reply:
https://github.com/theforeman/puppet-iop_advisor_engine/commit/5ae274d8aca70479f4f131c730b9d7c71a554fe5 just sneaked that in :)